test/testPartialPose3.jl

```diff
@@ -475,4 +475,53 @@ mu_fg2 = mean(M, mpts)
 @test_broken isapprox(submanifold_component(mu_fg2,1), [0,0,4], atol=0.2)
 
 ##
+end
+
+
+@testset "Test Pose3Pose3Rotation factor:" begin
+
+##
+# trivial cases first, orientation based tests below
+M3 = getManifold(Pose3)
+ϵ3 = getPointIdentity(Pose3)
+
+M2 = SpecialOrthogonal(3)
+ϵ2 = getPointIdentity(M2)
+
+ϕ = 0.1
+θ = 0.1
+ψ = 0.1
```

Comment:
Just a note these are exponentials, not quite RPY angles.  Perhaps we use alpha 1 2 3 or pqr.  Not serious, just a note.

Reply:
Jip, I copy and pasted from code that was updated from code that was RPY. with the caveat that only one angle is changed at a time. But you are correct.